man/sd_device_enumerator_new-example.c

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: MIT-0 */
+
+#include <systemd/sd-device.h>
+#include <stdio.h>
```

Comment:
Please sort headers alphabetically.
Also, I guess,
```c
#define _GNU_SOURCE 1
```
is necessary before includes.
Please see other example files.